packages/kolibri-components/src/KModal.vue

```diff
@@ -312,7 +307,8 @@
     position: fixed;
     top: 0;
     left: 0;
-    z-index: 24;
+    // z-index needs to be one less than .snackbar-backdrop
+    z-index: 15;
```

Comment:
I know @indirectlylit has informed me multiple times that elevations should align with [MD elevations](https://material.io/design/environment/elevation.html#default-elevations). It says snackbars are 6 and dialogs are 24. That doesn't really help but I don't know how to resolve the discrepancy.

Reply:
Hm, I think elevations aren't really z-indices, but actually correspond to actual elevations in display points and usually affected with box-shadow rules. I guess it's just a coincidence that the z-indices match up with the elevations.

Reply:
It's not a coincidence, no - but it's also not a Material spec

We chose by convention to align Kolibri z-indexes with Material elevations to people choosing arbitrary numbers just to get something above or below something else.

I think it's fine to have exceptions to that convention as long as we label them as-such, but in general let's try to stick to matching MD elevations.